### PR TITLE
metrics: compute tensorflow statistics

### PR DESCRIPTION
**Pull request by dborquez**
This PR computes average results for TF bench.
Additionally, it improves the data parsing from
all running containers.

Fixes: #7603

### CODE REVIEW

**Review by GabyCT — APPROVED** [review]

File: tests/metrics/machine_learning/tensorflow.sh [hunk 1]
```diff
@@ -59,7 +64,8 @@ function create_resnet_start_script() {
 
 cat <<EOF >>"${script}"
 #!/bin/bash
-python benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py -data_format=NHWC --device cpu --batch_size=${BATCH_SIZE} --num_batches=${NUM_BATCHES} > resnet_results
+pushd "benchmarks/scripts/tf_cnn_benchmarks"
+python tf_cnn_benchmarks.py -data_format=NHWC --device cpu --batch_size=${BATCH_SIZE} --num_batches=${NUM_BATCHES} > "/${RESNET_FILE}"
```
Comment (cmaf) [inline tests/metrics/machine_learning/tensorflow.sh:68]: Is there a reason this command was split into 2 lines?

File: tests/metrics/machine_learning/tensorflow.sh [hunk 2]
```diff
@@ -89,59 +98,60 @@ function tensorflow_test() {
...
-		check_file=$(sudo -E "${CTR_EXE}" t exec --exec-id "$(random_name)" "${i}" sh -c "${CMD_FILE}")
-		retries="300"
-		for j in $(seq 1 "${retries}"); do
-			[ "${check_file}" -eq "1" ] && break
-			sleep 1
+			# if the current task is done, remove the corresponding container from the active list
+			[ "${check_file}" -eq "1" ] && unset 'tasks_running[i]'
 		done
+		((retries--))
+		sleep 3
```
Comment (cmaf) [inline tests/metrics/machine_learning/tensorflow.sh:118]: This seems like a long time to sleep, why the increase?

**Review by cmaf — COMMENTED** [review]
Thanks @dborquez, two questions below

File: tests/metrics/machine_learning/tensorflow.sh (hunk 1 above)
Comment (dborquez) [reply tests/metrics/machine_learning/tensorflow.sh:68]: @cmaf thank you, I split the 'launch workload' line just to add clarity to the code

File: tests/metrics/machine_learning/tensorflow.sh (hunk 2 above)
Comment (dborquez) [reply tests/metrics/machine_learning/tensorflow.sh:118]: @cmaf thanks, the workload usually takes several minutes to complete (about 5 on my machine).
We added a "retries" variable as a timeout mechanism.

**Review by cmaf — APPROVED** [review]

**Pull request merged**